src/elm/Page/Community/ActionEditor.elm

```diff
@@ -1232,18 +1232,20 @@ viewValidations { shared } model =
                         , label [ class "form-switch-label", for "expiration-toggle" ] []
                         ]
                     , label [ class "flex text-body text-green", for "expiration-toggle" ]
-                        [ p [ class "font-bold mr-1" ]
-                            [ if model.form.validation == NoValidation then
-                                text_ "community.actions.form.validation_off"
+                        [ p [ class "mr-1" ] 
+                            [ Html.b [] 
```

Comment:
I think it's better to use just `b` since all other tags have no `Html.` prefix. You can add `b` in the `exposing` list in the line `36`.